### PR TITLE
A few fixes to the container-based install, which is broken on:

### PR DESCRIPTION
**Pull request by clkao**
* when pwd is /
* when stdin is not tty
* when invoking `run sh -c 'echo foo'`

### CODE REVIEW

File: script/run.sh [hunk 1]
```diff
@@ -31,7 +31,9 @@ fi
 
 
 # Setup volume mounts for compose config and context
-VOLUMES="-v $(pwd):$(pwd)"
+if [ $(pwd) != '/' ]; then
```
Comment (dnephin) [inline script/run.sh]: There should be double quotes around `"$(pwd)"` otherwise spaces will break this I think.

Comment (clkao) [inline script/run.sh]: ok, edited the commit and adding quotes to the -w in run as well.

**Comment by dnephin** [comment]
LGTM

**Comment by aanand** [comment]
LGTM

**Pull request merged**